### PR TITLE
Porting changes openconfig#312 to OTG location to same script.

### PR DESCRIPTION
**Pull request by stthummuri**

### CODE REVIEW

File: feature/bgp/addpath/otg_tests/route_propagation_test/route_propagation_test.go [hunk 1]
```diff
@@ -460,6 +466,11 @@ func TestBGP(t *testing.T) {
 			}
 
 			dut := ondatra.DUT(t, "dut")
+			t.Log("Configure Network Instance")
+			dutConfNIPath := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance)
+			dutConfNIPath.Type().Replace(t, telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+			dutConfNIPath.RouterId().Replace(t, dutPort2.IPv4)
```
Comment (jasdeep-hundal) [inline feature/bgp/addpath/otg_tests/route_propagation_test/route_propagation_test.go:472]: While router ID is a constant for these tests, keep it as part of the each test case like in the corresponding ATE test (https://github.com/openconfig/featureprofiles/blob/1a438857b49d6b423ff3a694912cd41a0b26afdf/feature/bgp/addpath/ate_tests/route_propagation_test/route_propagation_test.go#L194). The relevant DUT configuration for a particular test case is made more obvious that way.

**Review by jasdeep-hundal — CHANGES_REQUESTED** [review]

File: feature/bgp/addpath/otg_tests/route_propagation_test/route_propagation_test.go (hunk 1 above)
Comment (stthummuri) [reply feature/bgp/addpath/otg_tests/route_propagation_test/route_propagation_test.go]: Made the changes

**Comment by jasdeep-hundal** [comment]
Needs a gofmt.

**Comment by sachendras** [comment]
I got pinged by IXIA that the introduction of peer-group in https://github.com/openconfig/featureprofiles/pull/513/files broke the OTG version of RT-1.3. The KNE instance ran on Arista ceos-lab now seeing the gNMI error "found bad element: peer-group".
IMO, as per https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-bgp-peer-groups-peer-group, peer-group is a standard that should be supported by all implementations. I see similar requirement by Cisco as well in https://github.com/openconfig/featureprofiles/issues/391. Hence, support for the same should also exist in Arista ceos-lab.

**Pull request merged**